api/envoy/config/filter/http/cache/v2/cache.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.cache.v2;
```

Comment:
@htuch this should be v2alpha/v3alpha. Can we get the tooling to do that in place as soon as in reasonable? Should we track downgrading this to alpha once that is done? How should we track it?

Reply:
Tracked at https://github.com/envoyproxy/envoy/issues/9769, I'll work on this over the next couple of days.